content/maxwell1_fundamentals/maxwells_equations/maxwells_equations_in_homogeneous_media/dipole_sources/index.rst

```diff
@@ -7,43 +7,61 @@ Maxwell's Equations for Dipole Sources
     :file: ../../../../../underconstruction.html
 
 
-.. topic:: Purpose
+.. Purpose::
```

Comment:
can you please use the lower case `purpose` so it draws the box? 

![image](https://cloud.githubusercontent.com/assets/6361812/20008457/492af564-a266-11e6-9b8f-c003b384e76a.png)